client/src/components/button.scss

```diff
@@ -38,7 +38,7 @@
 }
 
 .button_clear:disabled {
-	@extend .button:disabled;
+	@extend .button,:disabled;
```

Comment:
Interesting. Was this broken somewhere? What does the `,` do here?

Reply:
I updated the package-lock.json for both client and server - actually I only left them out in the Dockerfile itself. I verified and fixed all issues by manually building the client and server. Might be work a second look after I fixed that - I didn't encouter any bugs so far with the Dashboard over the week.

Reply:
Sorry, just to be extra clear... this line adds a `,` between `.button` and `:disabled`. I'm trying to understand the reason for this change. Does this fix a bug with the button layout somewhere?